SourceX/DiabloUI/selhero.cpp

```diff
@@ -377,6 +377,8 @@ BOOL UiSelHeroMultDialog(
 	return UiSelHeroDialog(fninfo, fncreate, fnstats, fnremove, dlgresult, name);
 }
 
+#if 0 // Cowcat

```

Comment:
Let's not merge this before it compiles :)

Reply:
Include problems with <chronos>. A simple test for chronos in MOS works but not for warpos SDK.